lib/utils/constants.dart

```diff
@@ -1,5 +1,7 @@
 class Settings {
   static const String SERVER_URL = 'https://ofypets.indiepet.co.in/';
+  static const String WEB_URL = 'https://beta.ofypets.com/';
+  static const String CANCEL_URL = 'https://test.payu.in/cancel';
```

Comment:
Please add url  for production payubiz cancel action.

Reply:
Okay